### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by DesastreNatural**
changed user agent as explained in the open issue by Talim

### CODE REVIEW

**Comment by DesastreNatural** [comment]
#154

**Comment by DesastreNatural** [comment]
never did a pull request. I hope I did everything correctly

File: tgtg/__init__.py [hunk 1]
```diff
@@ -17,9 +17,10 @@
 ACTIVE_ORDER_ENDPOINT = "order/v6/active"
 INACTIVE_ORDER_ENDPOINT = "order/v6/inactive"
 USER_AGENTS = [
-    "TGTG/21.12.1 Dalvik/2.1.0 (Linux; U; Android 6.0.1; Nexus 5 Build/M4B30Z)",
-    "TGTG/21.12.1 Dalvik/2.1.0 (Linux; U; Android 7.0; SM-G935F Build/NRD90M)",
-    "TGTG/21.12.1 Dalvik/2.1.0 (Linux; Android 6.0.1; SM-G920V Build/MMB29K)",
+    # "TGTG/21.12.1 Dalvik/2.1.0 (Linux; U; Android 6.0.1; Nexus 5 Build/M4B30Z)",
+    # "TGTG/21.12.1 Dalvik/2.1.0 (Linux; U; Android 7.0; SM-G935F Build/NRD90M)",
+    # "TGTG/21.12.1 Dalvik/2.1.0 (Linux; Android 6.0.1; SM-G920V Build/MMB29K)",
+    "TGTG/22.2.1 Dalvik/2.1.0 (Linux; U; Android 9; SM-G955F Build/PPR1.180610.011)"
```
Comment (ahivert) [inline tgtg/__init__.py:23]: keep all previous lines and change only 21.12.1 to 22.2.1

**Review by ahivert — CHANGES_REQUESTED** [review]

**Comment by ahivert** [comment]
Thanks for this PR !

**Comment by jansmets** [comment]
I confirm this works

**Comment by jansmets** [comment]
Try bumping the version to 22.5.5

**Pull request merged**